docs/3.0/resources/upgrade-to-prefect-3.mdx

```diff
@@ -1,19 +1,28 @@
 ---
-title: Upgrade to Prefect 3
-description: Learn how to upgrade from Prefect 2 to Prefect 3.
+title: Upgrade to Prefect 3.0
+description: Learn how to upgrade from Prefect 2.x to Prefect 3.0.
 ---
 
-Prefect 3 introduces exciting new features and improvements while maintaining compatibility with most Prefect 2 workflows. For the majority of users, upgrading to Prefect 3 will be a seamless process that requires few or no code changes. This guide highlights key changes that you may need to consider when upgrading.
+Prefect 3.0 is now the default open source version!
+
+Prefect 3.0 introduces exciting new features and improvements while maintaining compatibility with most Prefect 2 workflows. 
```

Comment:
I'd like to avoid marketing language around releases where we can - so in this case, if we want to introduce 3.0 in some way, I think we need to say something concrete like 
```suggestion
Prefect 3.0 introduces a number of enhancements to the OSS product: a new events & automations backend for event-driven workflows and observability, improved runtime performance, autonomous task execution and a streamlined caching layer based on transactional semantics.

The majority of these enhancements maintain compatibility with most Prefect 2.0 workflows, but there are a few caveats that you may need to adjust for.
```

Reply:
Adding this change in local version and then pushing.